docs/Dependency-injection.md

```diff
@@ -81,36 +81,23 @@ public class MyRegistrar
         container.RegisterSingleton<IService, SomeService>();
 
         // Automapper
-        container.RegisterSingleton(() => GetMapper(container));
-    }
-
-    private AutoMapper.IMapper GetMapper(Container container)
-    {
-        var mp = container.GetInstance<MapperProvider>();
-        return mp.GetMapper();
+        container.RegisterSingleton(() => MapperProvider.GetMapper(container));
     }
 }
 
-public class MapperProvider
+public static class MapperProvider
 {
-    private readonly Container _container;
-
-    public MapperProvider(Container container)
-    {
-        _container = container;
-    }
-
-    public IMapper GetMapper()
+    public static IMapper GetMapper(Container container)
     {
         var mce = new MapperConfigurationExpression();
-        mce.ConstructServicesUsing(_container.GetInstance);
+        mce.ConstructServicesUsing(container.GetInstance);
```

Comment:
I think this line is not needed.

Reply:
That's not something I can't comment on. I'm not thst familiar with AutoMapper (just with Simlle Injector), so removing such line is not a decision I can make.

Reply:
Yes, but if you already have this code, please test that evertyhing still works without it. I'm unfamiliar with Simple Injector.

Reply:
That"s not up to me to test whether the code works without it. I'm just providing an updated version that works with v5 as well.

Reply:
Well, if you didn't test this code, why would we consider merging this PR? :)